src/components/ConfirmPopup.js

```diff
@@ -0,0 +1,33 @@
+import Popup from "./Popup";
+
+export default class ConfirmPopup extends Popup {
+  constructor(modalSelector) {
+    super(modalSelector);
+    this._form = this._popup.querySelector(".modal__container");
+    this._submitButton = this._form.querySelector(".confirmation__button");
+    this._submitbuttonTextContent = this._submitButton.textContent;
+  }
+
+  setSaving(isSaving) {
+    if (isSaving) {
+      this._submitButton.textContent = "Saving...";
```

Comment:
` "Saving..."` 

If you want to show a loading text while deleting, then it should be `Deleting...` here